lms/djangoapps/course_api/blocks/serializers.py

```diff
@@ -70,7 +77,6 @@ class BlockDictSerializer(serializers.Serializer):  # pylint: disable=abstract-m
     Serializer that formats a BlockStructure object to a dictionary, rather
     than a list, of blocks
     """
-    root = serializers.CharField(source='root_block_usage_key')
```

Comment:
We need to keep `root` in this serializer.  It indicates the `root` of the block structure.  When returning a `list` of blocks, using the `BlockSerializer`, the `root` and `blocks` fields do not exist.